pkg/report/table/misconfig.go

```diff
@@ -124,18 +124,18 @@ func (r *misconfigRenderer) renderSummary(misconf types.DetectedMisconfiguration
 		}
 	}
 
-	// severity
+	// ID & severity
 	switch misconf.Severity {
 	case severityCritical:
-		r.printf("<red><bold>%s: ", misconf.Severity)
+		r.printf("%s <red><bold>(%s): ", misconf.ID, misconf.Severity)
```

Comment:
It is better to use the AVDID field instead of ID because ID can contain aliases https://github.com/aquasecurity/trivy/blob/main/pkg/misconf/scanner.go#L452-L454

```bash
s3-bucket-logging (LOW): Bucket has logging disabled
══════════════════════════════════════════════════════════════════════════════════════════════════════════════════
Ensures S3 bucket logging is enabled for S3 buckets
```

Reply:
thanks! good catch, updated.